<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система подбора предметов для игры League of Legends</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="ruleBaseLabel">
          <property name="text">
           <string>База правил</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="rulesBaseTable"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="resultLabel">
          <property name="text">
           <string>Результаты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="result">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="rulesManagementLabel">
          <property name="text">
           <string>Управление правилами</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="addRuleBtn">
            <property name="text">
             <string>Добавить правило</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteRuleBtn">
            <property name="text">
             <string>Удалить правило</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadRulesBtn">
            <property name="text">
             <string>Загрузить правила</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveRulesBtn">
            <property name="text">
             <string>Сохранить правила</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="currentFactsLabel">
          <property name="text">
           <string>Известные факты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="currentFacts"/>
        </item>
        <item>
         <widget class="QPushButton" name="addFactBtn">
          <property name="text">
           <string>Добавить факт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteFactBtn">
          <property name="text">
           <string>Удалить факт</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="goalLabel">
          <property name="text">
           <string>Цель</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="goalInput"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="proveGoalBtn">
        <property name="text">
         <string>Доказать цель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="helpBtn">
        <property name="text">
         <string>Справка</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
